projects/debug_test/src/main.c

```diff
@@ -0,0 +1,69 @@
+#include <stdint.h>
```

Comment:
These `debug_test` and `hello_world` projects aren't part of the homework, please remove.